Source/WebCore/style/StyleTreeResolver.cpp

```diff
@@ -191,6 +191,8 @@ static bool affectsRenderedSubtree(Element& element, const RenderStyle& newStyle
 {
     if (newStyle.display() != DisplayType::None)
         return true;
+    if (element.hasDisplayNone())
+        return true;
```

Comment:
does `affectsRenderedSubtree` ever return `false` now?

Reply:
It does since making that function return `true` yielded a couple of test failures in the subset of tests I've tried this with locally:

`imported/w3c/web-platform-tests/css/cssom/getComputedStyle-display-none-001.html` → one new `PASS` result
`imported/w3c/web-platform-tests/css/css-transitions/starting-style-cascade.html` → one new `FAIL` result